File: app4web_05042021/app/src/main/java/site/app4web/app4web/Helper/JasonSettings.java
package site.app4web.app4web.Helper;

import android.content.Context;

import site.app4web.app4web.R;

public class JasonSettings {
    public static boolean isAsync(String className, Context context) {
        String[]asyncActions = context.getResources().getStringArray(R.array.asyncActions);
        for(String s: asyncActions){
            if(s.equalsIgnoreCase(className)) return true;
        }
        return false;
    }
}

File: app4web_28112021_kotlin/app/src/main/java/site/app4web/app4web/Helper/JasonSettings.kt
package site.app4web.app4web.Helper

import android.content.Context
import site.app4web.app4web.R


object JasonSettings {
    fun isAsync(className: String?, context: Context): Boolean {
        val asyncActions = context.resources.getStringArray(R.array.asyncActions)
        for (s in asyncActions) {
            if (s.equals(className, ignoreCase = true)) return true
        }
        return false
    }
}